subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -490,19 +508,22 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 	int opt;
 	int opt_index = 0;
 	struct getopt_state *state;
-	static const struct option long_options[] = {
-		{"ssid", required_argument, 0, 's'},
-		{"passphrase", required_argument, 0, 'p'},
-		{"key-mgmt", required_argument, 0, 'k'},
-		{"ieee-80211w", required_argument, 0, 'w'},
-		{"bssid", required_argument, 0, 'm'},
-		{"band", required_argument, 0, 'b'},
-		{"channel", required_argument, 0, 'c'},
-		{"timeout", required_argument, 0, 't'},
-		{"anon-id", required_argument, 0, 'a'},
-		{"key-passwd", required_argument, 0, 'K'},
-		{"help", no_argument, 0, 'h'},
-		{0, 0, 0, 0}};
+	static const struct option long_options[] = {{"ssid", required_argument, 0, 's'},
+						     {"passphrase", required_argument, 0, 'p'},
+						     {"key-mgmt", required_argument, 0, 'k'},
+						     {"ieee-80211w", required_argument, 0, 'w'},
+						     {"bssid", required_argument, 0, 'm'},
+						     {"band", required_argument, 0, 'b'},
+						     {"channel", required_argument, 0, 'c'},
+						     {"timeout", required_argument, 0, 't'},
+						     {"anon-id", required_argument, 0, 'a'},
+						     {"key-passwd", required_argument, 0, 'K'},
+						     {"suiteb-type", required_argument, 0, 'S'},
+						     {"eap-version", required_argument, 0, 'V'},
+						     {"eap-identity", required_argument, 0, 'I'},
+						     {"eap-password", required_argument, 0, 'P'},
```

Comment:
Could you add these to the option array without changing the alignment. These are now quite far right which is not really needed.

Reply:
Updated as comments.